contracts/contracts/eosio.system/src/voting.cpp

```diff
@@ -294,9 +294,10 @@ namespace eosiosystem {
       update_total_votepay_share( ct, -total_inactive_vpay_share, delta_change_rate );
 
       _voters.modify( voter, same_payer, [&]( auto& av ) {
-         av.last_vote_weight = new_vote_weight;
-         av.producers = producers;
-         av.proxy     = proxy;
+         av.last_vote_weight       = new_vote_weight;
```

Comment:
I would not recommend changing the default style and would keep the existing one with no additional spaces